apps/client/src/features/rundown/event-block/composite/EventBlockReporter.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tooltip } from '@chakra-ui/react';
+import { useDebouncedValue } from '@mantine/hooks';
+import { Playback, ReportData } from 'ontime-types';
+import { MILLIS_PER_MINUTE, MILLIS_PER_SECOND } from 'ontime-utils';
+
+import { useReportStatus } from '../../../../common/hooks/useSocket';
+import useReport from '../../../../common/hooks-query/useReport';
+import { formatDuration } from '../../../../common/utils/time';
+import { tooltipDelayFast } from '../../../../ontimeConfig';
+import { getTimeToStart } from '../../../../views/timeline/timeline.utils';
+
+import style from './EventBlockReporter.module.scss';
+
+export default function EventBlockReporter(props: { id: string; timeStart: number; isPastOrLoaded: boolean }) {
+  const { id, timeStart, isPastOrLoaded } = props;
+  const { data } = useReport();
+  const status = useReportStatus();
+  const [debouncedStatus] = useDebouncedValue(status, 200);
```

Comment:
What are you trying to achieve with this hook? The 200ms debounce is lower than the update rate of clock (1s)
Is this because the values are out of sync?
If that is the case, I don't think the clients should be patching up, and we rather that the server syncs up the state it sends 

Regardless:

We may want the leading flag on so we always get an update at start

I also think we want to isolate this in a separate hook. As it is, we are re-rendering the component in every change, even though we may not update the value we show